app/macos/Runner/Configs/AppInfo.xcconfig

```diff
@@ -0,0 +1,14 @@
+// Application-level settings for the Runner target.
+//
+// This may be replaced with something auto-generated from metadata (e.g., pubspec.yaml) in the
+// future. If not, the values below would default to using the project name when this becomes a
+// 'flutter create' template.
+
+// The application's name. By default this is also the title of the Flutter window.
+PRODUCT_NAME = prototype
+
+// The application's bundle identifier
+PRODUCT_BUNDLE_IDENTIFIER = com.example.prototype
```

Comment:
Could you please change it to `im.phnx.prototype`?

Reply:
Done, now it is

```
// The application's bundle identifier
PRODUCT_BUNDLE_IDENTIFIER = im.phnx.prototype

// The copyright displayed in application information
PRODUCT_COPYRIGHT = Copyright © 2025 Phoenix R&D GmbH. All rights reserved.
```